WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -133,6 +142,27 @@ extension PointOfSaleAggregateModel {
     }
 }
 
+// MARK: - Card payments
+
+extension PointOfSaleAggregateModel {
+    private func publishCardReaderConnectionStatus() {
```

Comment:
nit;
```suggestion
    private func observeCardReaderConnectionStatus() {
```

Reply:
I think publish is more appropriate here, no? The function is primarily to (re-)publish the value from the service, and the aggregate model doesn't (currently) care what that value is.